HW_3/task_3.py

```diff
@@ -0,0 +1,23 @@
+"""
+Задайте список из вещественных чисел, округленных до сотых.
+Найдите разницу между максимальным и минимальным значением дробной части элементов.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <float>
+
+Пример:
+[1.1, 1.2, 3.1, 5, 10.01]
+2.0
+"""
+import random
+
+number = int(input("Enter number: "))
+lst = [round(random.uniform(1, 9), 2) for i in range(number)]
+lst_2 = []
+
+for i in range(len(lst)):
```

Comment:
У вас ведь уже есть переменная, которая хранит длину
```suggestion
for i in range(number):
```